src/OrleansRuntime/Messaging/SiloMessageSender.cs

```diff
@@ -31,9 +31,9 @@ namespace Orleans.Runtime.Messaging
 {
     internal class SiloMessageSender : OutgoingMessageSender
     {
-        private readonly MessageCenter messageCenter;
+        protected readonly MessageCenter messageCenter;
         private const int DEFAULT_MAX_RETRIES = 0;
-        private readonly Dictionary<SiloAddress, DateTime> lastConnectionFailure;
+        protected readonly Dictionary<SiloAddress, DateTime> lastConnectionFailure;
```

Comment:
what's the goal in making these fields protected? I couldn't find any usages of them outside this class (and also I don't see any subclasses of SiloMessageSender.